pkg/doc/did/doc.go

```diff
@@ -345,3 +352,73 @@ func stringEntry(entry interface{}) string {
 	}
 	return entry.(string)
 }
+
+// JSONBytes converts document to json bytes
+func (doc *Doc) JSONBytes() ([]byte, error) {
+	raw := &rawDoc{
+		Context:        doc.Context,
+		ID:             doc.ID,
+		PublicKey:      populateRawPublicKeys(doc.PublicKey),
+		Authentication: populateRawAuthentications(doc.Authentication),
+		Service:        populateRawServices(doc.Service),
+		Created:        doc.Created,
+		Proof:          doc.Proof,
+		Updated:        doc.Updated,
+	}
+
+	byteDoc, err := json.Marshal(raw)
+	if err != nil {
+		return nil, errors.Errorf("Json unmarshalling of document failed: %w", err)
+	}
+
+	return byteDoc, nil
+}
+
+func populateRawServices(services []Service) []map[string]interface{} {
```

Comment:
What makes something "Raw"?

Reply:
Existing code has a struct named `rawDoc` - https://github.com/hyperledger/aries-framework-go/blob/master/pkg/doc/did/doc.go#L194. And many operations/variable names have `raw` as the prefix